cypress/integration/e2e/article.elements.spec.js

```diff
@@ -61,5 +61,19 @@ describe('Elements', function() {
 
             getIframeBody().contains('radiolab');
         });
+        it('should render the soundcloud embed', function() {
+            const getIframeBody = () => {
+                return cy
+                    .get('div[data-cy="soundcloud-embed"] > iframe')
+                    .its('0.contentDocument.body')
+                    .should('not.be.empty')
+                    .then(cy.wrap);
```

Comment:
do you need this?

Reply:
Yeah. See: https://www.cypress.io/blog/2020/02/12/working-with-iframes-in-cypress/